packages/bpk-svgs/gulpfile.babel.js

```diff
@@ -54,6 +54,15 @@ const largeIconPxSize = remToPx(largeIconSize);
 
 const colors = _(tokens.props)
   .filter({ category: 'colors', type: 'color' })
+  .filter(color => {
+    // We don't want to generate SVGs for colors that begine with Background or Line as these are not meant for icons.
+    // We want it for raw colors only.
+    if (color.name.startsWith('BACKGROUND') || color.name.startsWith('LINE')) {
+      return;
```

Comment:
I guess undefined is falsy? Should we be more explicit and return false?

Reply:
Yep that sounds good! Means I can also remove the eslint disable rule just below it too